composer.json

```diff
@@ -22,20 +22,19 @@
 	],
 	"license": "GPL-2.0-or-later",
 	"require": {
-		"php": ">= 7.4.0, <= 8.2.99",
+		"php": ">= 8.0.0, <= 8.2.99",
```

Comment:
Do we **really** need PHP 8 here? This extension targets TYPO3 v11 and it is supposed to work with PHP 7.4. I know that PHP 7.4 is deprecated. Yet, I'm unsure if we really need to require PHP 8 unless there's a good reason for it. The sole use-case in that patch is the `: static` return which is wrong as explained. And supporting TYPO3 v12 that requires PHP 8 is not a reason for enforcing PHP 8 in v11 as well, it won't be possible to install it together with TYPO3 v12 and PHP 7.4 anyway :)

Reply:
Nope, the `static` is not wrong, see above.

I did not raise because of this feature, but simply because this is a breaking release anyway, so we can work with current technology. (The other branch supports v11 and older PHP versions)